samples/QuickstartClient/Program.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Anthropic.SDK;
+using Microsoft.Extensions.AI;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using ModelContextProtocol.Client;
+using ModelContextProtocol.Protocol.Transport;
+
+var builder = Host.CreateEmptyApplicationBuilder(settings: null);
+
+builder.Configuration
+    .AddEnvironmentVariables()
+    .AddUserSecrets<Program>();
+
+var (command, arguments) = args switch
+{
+    [var script] when script.EndsWith(".py") => ("python", script),
+    [var script] when script.EndsWith(".js") => ("node", script),
+    [var script] when Directory.Exists(script) || (File.Exists(script) && script.EndsWith(".csproj")) => ("dotnet", $"run --project {script} --no-build"),
+    _ => ("dotnet", "run --project ../../../../QuickstartWeatherServer --no-build")

```

Comment:
What's the scenario this is enabling? Seems like it could be unnecessary complication for a getting started example, but I may be missing the impetus.

Reply:
Ok. Could you please add a comment about the purpose of this section?